src/main/java/com/thesis/innmanagement/controllers/ContractController.java

```diff
@@ -0,0 +1,59 @@
+package com.thesis.innmanagement.controllers;
+
+import com.thesis.innmanagement.exceptions.ResourceNotFoundException;
+import com.thesis.innmanagement.models.Contracts;
+import com.thesis.innmanagement.repositories.ContractRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@CrossOrigin(origins = "*")
+@RestController
+@RequestMapping("api/v1/contracts")
+public class ContractController {
+
+    @Autowired
+    private ContractRepository contractRepository;
+
+    @GetMapping("/")
+    public List<Contracts> GetAll(){
+        return contractRepository.findAll();
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<Contracts> GetContractById(@PathVariable(value = "id") Long id) throws ResourceNotFoundException {
+        Contracts contract = contractRepository.findById(id).orElseThrow(()->new ResourceNotFoundException("Contract not found on: " + id));
+        return ResponseEntity.ok().body(contract);
+    }
+
+    @PostMapping("/")
+    public Contracts Create(@Validated @RequestBody Contracts contract) throws Exception{
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Reply:
Thank you for your reviews, I will modify all base on Java conventions. Also, I am highly appreciated your help but this is my thesis so I will set it privately then. This is not because of fearing that you or others will point out my mistakes, which is the demand of thesis scope. Thank you and best wishes for you!
P/s: please reply me as soon as soon as possible so that I can set it privately right after that

Reply:
Thanks for your reply :)